### PR TITLE
Change behaviour of searchable/creatable Single Select to not use Chips as per design feedback

### PR DESCRIPTION
**Pull request by mrazian85**
[Link to updated components in Deploy Preview](https://deploy-preview-183--neo-react-library-storybook.netlify.app/?path=/story/components-select--searchable)

**Before tagging the team for a review, I have done the following:**
- [x] reviewed my code changes
- [x] ensured that all tests pass
- [x] updated the link at the top of this PR (or remove it if not applicable)
- [x] added any important information to this PR (description, images, GIFs, ect.)
- [x] tagged Matt if any visual changes have occurred
- [x] done an accessibility check on my work

While reviewing an unrelated issue with @matthewAvaya, he remarked that our Searchable/Creatable Single Selects should not be using Chips to display selected options, but instead function exactly like the basic Single Select and use text instead.

This PR adjusts the behavior of our Searchable/Creatable Single Select accordingly, and edits the test file as needed to match the new behavior.

### CODE REVIEW

File: src/components/Select/Searchable.test.jsx [hunk 1]
```diff
@@ -67,24 +67,6 @@ describe("Select", () => {
       const results = await axe(container);
       expect(results).toHaveNoViolations();
     });
-
```
Comment (mrazian85) [inline src/components/Select/Searchable.test.jsx:70]: Removed this test as our Single Select should not toggle selected option on click

File: src/components/Select/Searchable.test.jsx [hunk 2]
```diff
@@ -158,18 +140,17 @@ describe("Select", () => {
...
 
-      // newly created chip has been added
-      expect(screen.getAllByRole("button")).toHaveLength(1);
-      expect(screen.getByText(newOptionText)).toBeInTheDocument();
+      // newly created text has been added
+      expect(queryByText(newOptionText)).toBeTruthy();
 
       await act(async () => await user.keyboard(UserEventKeys.BACKSPACE));
-      expect(screen.queryAllByRole("button")).toHaveLength(0); // chip removed
+      expect(queryByText(newOptionText)).toBeFalsy(); // text removed
```
Comment (joe-s-avaya) [inline src/components/Select/Searchable.test.jsx:147]: why are you not using `screen.getAllByText`?

Comment (joe-s-avaya) [reply src/components/Select/Searchable.test.jsx:147]: Oh wait, it is because `get*` explodes if it's not found? Use `screen.queryAllByText`.

**Review by matthewAvaya — APPROVED** [review]

**Review by yangAtSpoken — COMMENTED** [review]
Height of Select seems way bigger. Do you want to fix it here or in a new story?

**Comment by mrazian85** [comment]
> Height of Select seems way bigger. Do you want to fix it here or in a new story?

Great catch; it seems related to the `padded-container` class. I think I can fix it in this PR

**Review by joe-s-avaya — APPROVED** [review]

**Review by yangAtSpoken — APPROVED** [review]

**Pull request merged**